qemu/tests/bridge_vlan.py

```diff
@@ -139,6 +139,7 @@ def netperf_vlan(client='main_vm', server='localhost',
                               "server: %s, client: %s" % (server, client),
                               test.log.info)
         session.cmd("systemctl restart NetworkManager", ignore_all_errors=True)
+        utils_net.restart_guest_network(session)
```

Comment:
The `session` always change during the whole test, suggest use `sessions[x]` instead it like `ping_vlan`, L141 has the same issue.

Reply:
I think using session directly is also a good option, because it is not exactly the same as ping_vlan, it needs to test netperf from different guest to host. This is the same as the case of ping_vlan on line 212. Please correct me if I'm wrong.

Reply:
https://github.com/autotest/tp-qemu/blob/d380199d678f9915471fb481bc0b96157bd392cd/qemu/tests/bridge_vlan.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L251-L253
These lines are out of the `for` loop, if you use the session directly, can you ensure it's the correct session you want to use?

Reply:
Because during the test, both VMs will be set to temporary private IP addresses, so both need to get public IP addresses before the netperf test. So I think there is no need to distinguish specific session here.

Reply:
So which session will be used in the `netperf_vlan`?

Reply:
Both guest sessions will be used, so I don't think it's necessary to differentiate it.

Reply:
"sessions[vm_index]" seems like a good choice if case need to specify sessions in tests, but from my point of view, there's not much difference between the two. Both will execute commands on both guests. Please correct me if I'm wrong.

Reply:
One session can only execute the command in one guest, can you explain in L253, https://github.com/autotest/tp-qemu/blob/d380199d678f9915471fb481bc0b96157bd392cd/qemu/tests/bridge_vlan.py?_pjax=%23js-repo-pjax-container%2C%20div%5Bitemtype%3D%22http%3A%2F%2Fschema.org%2FSoftwareSourceCode%22%5D%20main%2C%20%5Bdata-pjax-container%5D#L251-L253
`utils_net.restart_guest_network(session)` how to execute twice in both server and client?

Reply:
@PaulYuuu Thank you for your repeated corrections to this question. it is not a good way to add it here in this form. It will cause confusion about the use of session. After thinking about it, I decided to put this line before line 242.

Reply:
@PaulYuuu Please help me review it again, thanks a lot.